cdn/build-cdn/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "@talend/scripts-build-cdn",
+  "version": "0.0.0",
```

to be consistent with the script version
```suggestion
  "version": "0.0.1",
```